### PR TITLE
Bump eve-api

### PR DESCRIPTION
**Pull request by christoph-zededa**
Need to bump eve-api after following PR
https://github.com/lf-edge/eve-api/pull/70

### CODE REVIEW

File: pkg/pillar/go.sum [hunk 1]
```diff
@@ -1428,6 +1428,8 @@ github.com/lf-edge/edge-containers v0.0.0-20240207093504-5dfda0619b80 h1:kiqB1Rk
 github.com/lf-edge/edge-containers v0.0.0-20240207093504-5dfda0619b80/go.mod h1:4yXdumKdTzF0URMtxOl8Xnzdxnoy1QR+2dzfOr4CIZY=
 github.com/lf-edge/eve-api/go v0.0.0-20240829123634-7c8ebda876ff h1:3uGTOvWQFQkIrlkFalmzUmXINnzmVOAn5Zx0ryBSzxQ=
 github.com/lf-edge/eve-api/go v0.0.0-20240829123634-7c8ebda876ff/go.mod h1:ot6MhAhBXapUDl/hXklaX4kY88T3uC4PTg0D2wD8DzA=
+github.com/lf-edge/eve-api/go v0.0.0-20241125213346-b026f21b3e75 h1:vEu+sJLajrsVrBOKSl9+rZZ2tD6kg4Pi4gGiNAT2Cx0=
```
Comment (OhmSpectator) [inline pkg/pillar/go.sum:1431]: I'm just curious, why wasn't it here before...

**Review by OhmSpectator — APPROVED** [review]
run the test

File: pkg/pillar/go.sum [hunk 2]
```diff
@@ -1428,6 +1428,8 @@ github.com/lf-edge/edge-containers v0.0.0-20240207093504-5dfda0619b80 h1:kiqB1Rk
 github.com/lf-edge/edge-containers v0.0.0-20240207093504-5dfda0619b80/go.mod h1:4yXdumKdTzF0URMtxOl8Xnzdxnoy1QR+2dzfOr4CIZY=
 github.com/lf-edge/eve-api/go v0.0.0-20240829123634-7c8ebda876ff h1:3uGTOvWQFQkIrlkFalmzUmXINnzmVOAn5Zx0ryBSzxQ=
 github.com/lf-edge/eve-api/go v0.0.0-20240829123634-7c8ebda876ff/go.mod h1:ot6MhAhBXapUDl/hXklaX4kY88T3uC4PTg0D2wD8DzA=
```
Comment (milan-zededa) [inline pkg/pillar/go.sum:1430]: The previous version should go away If you run `go mod tidy`

File: pkg/pillar/go.sum (hunk 1 above)
Comment (milan-zededa) [reply pkg/pillar/go.sum:1431]: This is the new version of eve-api that we are upgrading to.

File: pkg/pillar/go.sum (hunk 2 above)
Comment (christoph-zededa) [reply pkg/pillar/go.sum]: Thx. Done.

**Review by milan-zededa — APPROVED** [review]

File: pkg/pillar/go.sum (hunk 1 above)
Comment (OhmSpectator) [reply pkg/pillar/go.sum:1429]: So, should we do the same here?..

Comment (OhmSpectator) [reply pkg/pillar/go.sum:1429]: Ah, I see

**Pull request merged**